src/Streamly/Internal/System/Process.hs

```diff
@@ -382,7 +381,7 @@ pipeChunksWithAction ::
     -> [String]             -- ^ Arguments
     -> Stream m a     -- ^ Output stream
 pipeChunksWithAction run modCfg path args =
-    Stream.bracket'
+    Stream.bracket3IO
```

Comment:
Remove liftIO from alloc, cleanupNormal, cleanupException, the type of these can be IO now.